### DISCUSSION TITLE
Extending documentation in xunit area for recipe how to use Reqnroll ITestOutputHelper and Alba when testing WebApi

### DISCUSSION THREAD

**Discussion by AdaskoTheBeAsT** [reqnroll/Reqnroll#226] [Contributor Q&A]
Hi,

I would like to extend documentation for xunit area for recipe how to use Reqnroll ITestOutputHelper and Alba
I have some small working recipe which can help also other people

**Comment by gasparnagy** [comment]
This is great. You can either add your description as a new page to the documentation to the [integrations](https://github.com/reqnroll/Reqnroll/tree/main/docs/integrations) section or if that is more like a sample, then you can just link it from the [samples section](https://github.com/reqnroll/Reqnroll/blob/main/docs/help/samples.md).

Please let us know if you need help (with review or with linking it to the docs).
